index.js

```diff
@@ -40,4 +40,9 @@ NFC.addListener = (callback) => {
     _registerToEvents();
 };
 
-export default NFC;
\ No newline at end of file
+NFC.removeListener = () => {
+    DeviceEventEmitter.removeListener(NFC_DISCOVERED);
+    _listeners.splice(0, _listeners.length);
+    _registeredToEvents = false;
```

Comment:
_registeredToEvents  keeps track if the DeviceEventEmitter.addListener is still attached. 
This only needs to be set to false if DeviceEventEmitter is removed. 
I created a removeAllListeners method because this truly removes the eventListener.
Perhaps I should call `removeAllListeners` `destroy`  instead?